Sources/StreamChatUI/ChatChannelList/ChatChannelListVC.swift

```diff
@@ -169,6 +173,50 @@ open class _ChatChannelListVC<ExtraData: ExtraDataTypes>: _ViewController,
             setupParentNavigation(parent: parent)
         }
     }
+
+    public func willShowActionViews(for indexPath: IndexPath) {
+        // Close other open cells
+        collectionView.visibleCells.forEach {
+            let cell = ($0 as? _ChatChannelListCollectionViewCell<ExtraData>)
+            cell?.swipeableView.close()
+        }
+
+        Animate { self.collectionView.layoutIfNeeded() }
+    }
+
+    public func actionViews(for indexPath: IndexPath) -> [UIView] {
+        let deleteButton = SwipeActionButton().withoutAutoresizingMaskConstraints
+
+        deleteButton.setImage(uiConfig.images.messageActionDelete, for: .normal)
+
+        deleteButton.backgroundColor = uiConfig.colorPalette.alert
+        deleteButton.tintColor = .white
+
+        deleteButton.action = {
+            // Add action for delete action.
+            print("Delete action button pressed")
+        }
```

Comment:
It would be great to forward this to some method users can subclass. Maybe something like:
```
open func deleteButtonPressedForCell(at indexPath: IndexPath)
```

Reply:
Added.. Implementation will be later...